src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/CsmSiteRecoveryEntity.cs

```diff
@@ -29,41 +27,41 @@ public partial class CsmSiteRecoveryEntity
         /// <summary>
         /// Initializes a new instance of the CsmSiteRecoveryEntity class.
         /// </summary>
-        public CsmSiteRecoveryEntity(DateTime? snapshotTime = default(DateTime?), bool? recoverConfig = default(bool?), string siteName = default(string), string slotName = default(string))
+        /// <param name="snapshotTime">Point in time in which the app recovery
+        /// should be attempted.</param>
+        /// <param name="siteName">[Optional] Destination app name into which
+        /// app should be recovered. This is case when new app should be
+        /// created instead.</param>
+        /// <param name="slotName">[Optional] Destination app slot name into
+        /// which app should be recovered.</param>
+        public CsmSiteRecoveryEntity(System.DateTime? snapshotTime = default(System.DateTime?), string siteName = default(string), string slotName = default(string))
         {
             SnapshotTime = snapshotTime;
-            RecoverConfig = recoverConfig;
             SiteName = siteName;
             SlotName = slotName;
         }
 
         /// <summary>
-        /// Point in time in which the site recover should be attempted.
+        /// Gets or sets point in time in which the app recovery should be
+        /// attempted.
         /// </summary>
         [JsonProperty(PropertyName = "snapshotTime")]
-        public DateTime? SnapshotTime { get; set; }
-
-        /// <summary>
-        /// If true, then the website's configuration will be reverted to its
-        /// state at SnapshotTime
-        /// </summary>
-        [JsonProperty(PropertyName = "recoverConfig")]
-        public bool? RecoverConfig { get; set; }
```

Comment:
This is fine because it's for a feature that isn't announced yet. I wanted to remove this property before documenting the feature.